oaknational/Oak-Web-Application
# fix: correct subject fliters on search page LESQ-485

## Description

Music year: 2018

- [ ]  Remove Physics filter from old search api subject list
- [ ]  Fetch a list of unique subjects from both api’s

## Issue(s)

Fixes # LESQ-485

## How to test

1. Go to https://deploy-preview-2136--oak-web-application.netlify.thenational.academy/teachers/search
You should see a complete list of filter subject from new and old content.

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
